File: 225. Implement Stack using Queues/MyStack.java
class MyStack {
    private int size;
    private int[] arr;
    private int top;

    public MyStack() {
        top = -1;
        size = 1000;
        arr = new int[size];
    }

    public void push(int x) {
        top++;
        arr[top] = x;
    }

    public int pop() {

        int x = arr[top];
        top--;
        return x;
    }

    public int top() {

        return arr[top];
    }

    public boolean empty() {
        return top == -1;
    }
}

/**
 * Your MyStack object will be instantiated and called as such:
 * MyStack obj = new MyStack();
 * obj.push(x);
 * int param_2 = obj.pop();
 * int param_3 = obj.top();
 * boolean param_4 = obj.empty();
 */

File: 225. Implement Stack using Queues/MyStack.cpp
class MyStack
{
    int size;
    int *arr;
    int topp;

public:
    MyStack()
    {
        topp = -1;
        size = 1000;
        arr = new int[size];
    }

    void push(int x)
    {
        topp++;
        arr[topp] = x;
    }

    int pop()
    {
        int x = arr[topp];
        topp--;
        return x;
    }

    int top()
    {
        return arr[topp];
    }

    bool empty()
    {
        return topp == -1 ? true : false;
    }
};

/**
 * Your MyStack object will be instantiated and called as such:
 * MyStack* obj = new MyStack();
 * obj->push(x);
 * int param_2 = obj->pop();
 * int param_3 = obj->top();
 * bool param_4 = obj->empty();
 */